transportccextension.go

```diff
@@ -0,0 +1,49 @@
+package rtp
+
+import (
+	"encoding/binary"
+	"errors"
+)
+
+const (
+	// TransportCCExtensionSize 4 byte
+	TransportCCExtensionSize = 4
+)
+
+var (
+	errTooSmall = errors.New("buffer too small")
+)
+
+// TransportCCExtension is a extension payload format in
+// https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01
```

Comment:
Will this format weird in the godoc?

The byte layout is important to have but I'm not sure it belongs in tbe docstring